admin-site/src/Login.svelte

```diff
@@ -20,28 +20,29 @@
 
  
 // add this to the above line to keep track of serverUrl value in store: , urlStore.subscribe(value => {console.log(value)}) 
-  onMount(async () => {
+  // onMount(async () => {
```

Comment:
If you don't need it, kill it (or save it in a gist / in your notes)

Reply:
Gone